<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>677</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
   </font>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon/collect.png</normaloff>:/icon/collect.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : rgb(238, 238, 236)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolButton" name="in_out_account">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Abyssinica SIL</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;in_out&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color : rgb(85, 170, 255);
border:2px groove gray;border-radius:30px;padding:2px 4px;
border-width: 3px;
border-style: solid;
border-color: darkblue;</string>
          </property>
          <property name="text">
           <string>出入户</string>
          </property>
          <property name="icon">
           <iconset resource="icon.qrc">
            <normaloff>:/icon/people.png</normaloff>:/icon/people.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>96</width>
            <height>96</height>
           </size>
          </property>
          <property name="popupMode">
           <enum>QToolButton::DelayedPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="account_card_borrow">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Abyssinica SIL</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color : rgb(85, 170, 255);
border:2px groove gray;border-radius:30px;padding:2px 4px;
border-width: 3px;
border-style: solid;
border-color: darkblue;</string>
          </property>
          <property name="text">
           <string>借卡</string>
          </property>
          <property name="icon">
           <iconset resource="icon.qrc">
            <normaloff>:/icon/credit_card_172.49661399549px_1229314_easyicon.net.png</normaloff>:/icon/credit_card_172.49661399549px_1229314_easyicon.net.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>96</width>
            <height>96</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="readCardButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color : rgb(85, 170, 255);
border:2px groove gray;border-radius:30px;padding:2px 4px;
border-width: 3px;
border-style: solid;
border-color: darkblue;</string>
          </property>
          <property name="text">
           <string>绑卡</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icon/readCard.png</normaloff>icon/readCard.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>96</width>
            <height>96</height>
           </size>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="search">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Abyssinica SIL</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color : rgb(85, 170, 255);
border:2px groove gray;border-radius:30px;padding:2px 4px;
border-width: 3px;
border-style: solid;
border-color: darkblue;</string>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
          <property name="icon">
           <iconset resource="icon.qrc">
            <normaloff>:/icon/search_128px_1232043_easyicon.net.png</normaloff>:/icon/search_128px_1232043_easyicon.net.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>96</width>
            <height>96</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>4</number>
        </property>
        <widget class="QWidget" name="stackedWidgetPage0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="Login_page" name="log_in_page">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="stackedWidgetPage1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="IN_OUT_ACCOUNT_PAGE" name="in_out_account_page">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="stackedWidgetPage2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="ACCOUNT_CARD_BORROW_PAGE" name="account_card_borrow_page">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="stackedWidgetPage3">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="SEARCH_PAGE" name="search_page">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="stackedWidgetPage4">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="RecordCardNumber" name="readCardPage" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>IN_OUT_ACCOUNT_PAGE</class>
   <extends>QFrame</extends>
   <header>in_out_account_page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ACCOUNT_CARD_BORROW_PAGE</class>
   <extends>QFrame</extends>
   <header>account_card_borrow_page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SEARCH_PAGE</class>
   <extends>QFrame</extends>
   <header>search_page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Login_page</class>
   <extends>QFrame</extends>
   <header>login_page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RecordCardNumber</class>
   <extends>QWidget</extends>
   <header>recordcardnumber.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
